tests/ami/scripts/test_windows.ps1.j2

```diff
@@ -71,6 +71,10 @@ function GetMSIVersion {
     }
 }
 
+# Add env.variable to force the Python interpreter to work with unicode. We also make it system-wide to have an effect
+# when it is started as a service.
+[Environment]::SetEnvironmentVariable("PYTHONUTF8", "1", "Machine")
```

Comment:
Sorry, to clarify - I mean if there is a way to set this as a default for the agent process to avoid additional manual intervention on the end user side (e.g. in the agent batch start up script or similar).

As discussed on the call, if that's not possible, we should see if we can create a knowledge base article on how to do that + update log message which is logged by the agent to point to that KB article in case that environment variable is not set.